src/DynamoSandbox/DynamoCoreSetup.cs

```diff
@@ -56,36 +62,21 @@ public void RunApplication(Application app)
         {
             try
             {
-                DynamoModel.RequestMigrationStatusDialog += MigrationStatusDialogRequested;
-                DynamoModel model;
-                Dynamo.Applications.StartupUtils.ASMPreloadFailure += ASMPreloadFailureHandler;
-                model = Dynamo.Applications.StartupUtils.MakeModel(false, ASMPath ?? string.Empty, analyticsInfo);
-
-                model.CERLocation = CERLocation;
-
-                viewModel = DynamoViewModel.Start(
-                    new DynamoViewModel.StartConfiguration()
-                    {
-                        CommandFilePath = commandFilePath,
-                        DynamoModel = model,
-                        Watch3DViewModel =
-                            HelixWatch3DViewModel.TryCreateHelixWatch3DViewModel(
-                                null,
-                                new Watch3DViewModelStartupParams(model),
-                                model.Logger),
-                        ShowLogin = true
-                    });
+                DynamoModel.RequestUpdateLoadBarStatus += DynamoModel_RequestUpdateLoadBarStatus;
 
-                var view = new DynamoView(viewModel);
-                view.Loaded += OnDynamoViewLoaded;
+                migrationWindow = new SettingsMigrationWindow();
+                migrationWindow.webView.NavigationCompleted += WebView_NavigationCompleted;
+                migrationWindow.RequestLaunchDynamo = LaunchDynamo;
+                migrationWindow.RequestImportSettings = ImportSettings;
+                migrationWindow.Show();
```

Comment:
@filipeotero @reddyashish @zeusongit I notice that we may not have time to adjust this before @filipeotero 's vacation, but I think we need to rename this view and move it back to DynamoCoreWpf project instead of maintaining it as part of Sandbox project. The reason I kept wondering is in order to have the splash screen to appear in DynamoRevit context